src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebLogsRouteConfig.kt

```diff
@@ -0,0 +1,18 @@
+package org.vaccineimpact.orderlyweb.app_start.routing.web
+
+import org.vaccineimpact.orderlyweb.*
+import org.vaccineimpact.orderlyweb.app_start.RouteConfig
+import org.vaccineimpact.orderlyweb.controllers.web.LogsController
+
+object WebLogsRouteConfig : RouteConfig
```

Comment:
Our existing report-running routes are all in `WebReportRouteConfig`, so this endpoint should probably be there. Or in a new `WebReportRunConfig` with the others... though that's a job worthy of it's own ticket!

Reply:
From my initial discussions with @EmmaLRussell we had decided that reportLogs should be separate from reportRun